graphql/util.go

```diff
@@ -58,40 +58,60 @@ func getGroupedFiles(ctx context.Context, name string, taskID string, execution
 	return &GroupedFiles{TaskName: &name, Files: apiFileList}, nil
 }
 
-func setScheduled(ctx context.Context, url, taskID string, isActive bool) (*restModel.APITask, error) {
+func setManyTasksScheduled(ctx context.Context, url string, isActive bool, taskIDs ...string) ([]*restModel.APITask, error) {
 	usr := mustHaveUser(ctx)
-	t, err := task.FindOneId(taskID)
+	tasks, err := task.FindAll(db.Query(task.ByIds(taskIDs)))
 	if err != nil {
 		return nil, ResourceNotFound.Send(ctx, err.Error())
 	}
-	if t == nil {
-		return nil, ResourceNotFound.Send(ctx, errors.Errorf("task %s not found", taskID).Error())
+	if len(tasks) != len(taskIDs) {
+		grip.Error(message.Fields{
+			"message":    "could not find all tasks",
+			"taskIds":    taskIDs,
+			"foundTasks": tasks,
+		})
+		return nil, ResourceNotFound.Send(ctx, errors.New("tasks not found").Error())
+	}
+	catcher := grip.NewBasicCatcher()
+	for _, t := range tasks {
+		if t.Requester == evergreen.MergeTestRequester && isActive {
+			catcher.Add(InputValidationError.Send(ctx, "commit queue tasks cannot be manually scheduled"))
```

Comment:
I may have been wrong suggesting a catcher in this case considering Mohamed's comment that we would expect all of the tasks in the same version to have the same requester. We may want to just return the error in this case.